docs/governance/templates/release_issue.md

```diff
@@ -26,43 +26,55 @@ There are 3 types of releases:
 1. Close the release issue.
 
 !-->
+
+Create a Branch
+---------------
+**These instructions are for the first release candidate of a minor (X.Y) release.**
+- [ ] Create a branch in the upstream repository. It should be named release-X.Y. Example: release-0.5. At this point there's effectively a code freeze for this version and all work on master will be included in a future version.
+
+Create New Version
+------------------
+_Find the release branch relevant for your release. It will have the name release-X.Y. While doing a release in the branch no changes should be submitted outside of the process._
+
+- [ ] Announce a PR freeze on release-X.Y branch on [open-match-discuss@](mailing-list-post).
+- [ ] Create a PR to bump the version.
+  - [ ] Open the [`Makefile`](makefile-version) and change BASE_VERSION value. Release candidates use the -rc.# suffix.
+  - [ ] Open the [`install/helm/open-match/Chart.yaml`](om-chart-yaml-version) and [`install/helm/open-match-example/Chart.yaml`](om-example-chart-yaml-version) and change the `appVersion` and `version` entries.
+  - [ ] Open the [`install/helm/open-match/values.yaml`](om-values-yaml-version) and [`install/helm/open-match-example/values.yaml`](om-example-values-yaml-version) and change the `tag` entries.
+  - [ ] Open the [`site/config.toml`]  and change the `release_branch` and `release_version` entries.
+  - [ ] Open the [`README.md`](readme-deploy) update the version references.
```

Comment:
This should be moved to one of the last steps, and be excluded for RCs.  This readme won't be on the main github page because it's never commited to main.